.github/workflows/tests_and_checks.yml

```diff
@@ -30,13 +30,18 @@ jobs:
         run: |
           poetry run pytest tests -m unit
 
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v3
-      - run: npm install --save-dev hardhat
+      - name: Set up node
+        uses: actions/setup-node@v3.0.0
+        with:
+          node-version: 'lts/*'
```

Comment:
I think better to have specific version, like `18.17.1`

Reply:
Hm, I took example from keys-api

Reply:
Fixed